results/cmd/api/results.sql

```diff
@@ -1,2 +1,4 @@
+-- logid serve as auto-generated key by sqltite to uniquely identify a taskrun log
+-- uid is the key from taskrun itself we now use to do query on taskrun logs, future we may use cluster-info as key
```

Comment:
Which field from the taskrun does this correspond to? Should this be a string instead?

Reply:
Yes.

Reply:
done